src/engine/engine-manager.ts

```diff
@@ -74,18 +74,14 @@ export class IvyEngineManager {
     });
   }
 
-  public async initProjects() {
-    if (await this.devContextPath) {
-      const ivyProjectDirectories = await this.ivyProjectDirectories();
-      await this.ivyEngineApi.initProjects(ivyProjectDirectories);
-    }
+  private async initProjects(): Promise<void> {
```

Comment:
no need to define return statement. Should be infered from the content of this function